deploy.php

```diff
@@ -74,7 +74,11 @@
 task(
     'build:assets:npm',
     function () {
-        run('npm run build');
+        if (commandExist('nvm')) {
+            run('nvm exec npm run build');
```

Comment:
You should run `nvm install` before this. If nvm detects a version that isn't installed yet, it will throw an error upon running `nvm exec ..`

Reply:
Fixed in e6826ba2f94e5b90e44e120ec018d326e107c575